pkgs/development/tools/jless/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "jless";
+  version = "0.7.1";
+
+  src = fetchFromGitHub {
+    owner = "PaulJuliusMartinez";
+    repo = "jless";
+    rev = "v${version}";
+    sha256 = "sha256-gBqyo/N/qF6HCTUrSKNVLiL1fc/JTfip1kNpNCBzRT8=";
+  };
+
+  cargoSha256 = "sha256-PbX61RVbrI2kTuyXK+LhQdJDvNo3KjIQH5eBbL6iUBM=";
+
+  doCheck = false;
```

Comment:
If it is only a couple of tests that fail they can be selectively disabled via `checkFlags`.

https://github.com/NixOS/nixpkgs/blob/00be9c5821622fefc6da17a664b1a536a5cbe5f7/pkgs/tools/misc/miniserve/default.nix#L28-L31

If a lot of the tests fail `doCheck = false` is fine but please add a comment.

Reply:
Good catch. This was actually based off of another similar derivation and I forgot to remove `doCheck = false`. The tests actually all pass so I removed the line. Thanks!